### PR TITLE
Refactor integration test package to allow it consumed by others, like performance test

### PR DESCRIPTION
**Pull request by chaodaiG**
/cc @stati0n

### CODE REVIEW

File: internal/integration/integration.go [hunk 1]
```diff
@@ -31,8 +31,30 @@ import (
...
+	authorizedappmodel "github.com/google/exposure-notifications-server/internal/authorizedapp/model"
+	exportdatabase "github.com/google/exposure-notifications-server/internal/export/database"
+	exportmodel "github.com/google/exposure-notifications-server/internal/export/model"
+)
+
+const (
+	exportDir = "my-bucket"
 )
 
+func TestServer(tb testing.TB, ctx context.Context, exportPeriod time.Duration) (*serverenv.ServerEnv, *EnServerClient, *database.DB) {
```
Comment (sethvargo) [inline internal/integration/integration.go:44]: We should make this NewTestServer or something more explicit

Comment (chaodaiG) [reply internal/integration/integration.go]: Agree and updated

File: internal/integration/integration.go [hunk 2]
```diff
@@ -31,8 +31,31 @@ import (
...
+	exportmodel "github.com/google/exposure-notifications-server/internal/export/model"
+)
+
+const (
+	exportDir = "my-bucket"
 )
 
+// NewTestServer sets up clients used for integration tests
+func NewTestServer(tb testing.TB, ctx context.Context, exportPeriod time.Duration) (*serverenv.ServerEnv, *EnServerClient, *database.DB) {
+	env, client := testServer(tb)
```
Comment (stati0n) [inline internal/integration/integration.go:46]: Does this need to be `TestServer`?

Comment (stati0n) [reply internal/integration/integration.go:46]: Never mind.  Think I got something wrong when I checked out and merged your branch ;)

Comment (chaodaiG) [reply internal/integration/integration.go:46]: :)

**Pull request merged**